app/patient-details/views/patient-arv-pickup-history-report.html

```diff
@@ -3,97 +3,111 @@
+      <div id="calendar-conteiner">
+        <h2>{{'DISPENSATION_CONTROL_CALENDAR' | translate}}</h2>
+        <table id="calendar">
+          <tr>
+            <th class="empty"></th>
+            <th class="empty"></th>
+            <th ng-repeat="month in months track by $index">{{month}}</th>
+          </tr>
+          <tr>
+            <th class="long-side-header" rowspan="5">Semana</th>
+            <th>{{'FIRST_ABRV' | translate}}</th>
+            <td ng-repeat="x in calendar[0] track by $index">{{x}}</td>
+          </tr>
+          <tr>
+            <th>{{'SECOND_ABRV' | translate}}</th>
+            <td ng-repeat="x in calendar[1] track by $index">{{x}}</td>
+          </tr>
+          <tr>
+            <th>{{'THIRD_ABRV' | translate}}</th>
+            <td ng-repeat="x in calendar[2] track by $index">{{x}}</td>
+          </tr>
+          <tr>
+            <th>{{'FORTH_ABRV' | translate}}</th>
+            <td ng-repeat="x in calendar[3] track by $index">{{x}}</td>
           </tr>
-          </tbody>
         </table>
       </div>
-    </div>
   </div>
 
-  <div class="row">
-    <div class="col-xs-10 col-xs-offset-1">
-      <h2>Levantamentos</h2>
-      <div class="pickups">
-        <table class="table table-bordered table-striped">
-          <thead>
-            <tr>
-              <th>{{'PHARMACY_PICKUP_DATE' | translate}}</th>
-              <th>{{'PHARMACY_REGIMEN'     | translate}}</th>
-              <th>{{'PHARMACY_QUANTITY_SHORT' | translate}}</th>
-              <th>{{'PHARMACY_POSOLOGY'    | translate}}</th>
-              <th>{{'PHARMACY_NEXT_PICKUP' | translate}}</th>
-            </tr>
-          </thead>
-          <tbody>
-            <tr ng-repeat="p in patient.pickups">
-              <td>{{p.encounterDatetime | date}}</td>
-              <td>{{p.regimen}}</td>
-              <td>{{p.quantity}}</td>
-              <td>{{p.posology}}</td>
-              <td>{{p.nextPickup  | date}}</td>
-            </tr>
-          </tbody>
-        </table>
+  <div id="top-right-area">
+    <h2 id="top-right-area-title">{{'FILA_REPORT_PERSONAL_DATA_NOTE' | translate}}</h2>
+    <div id="patient-data-container">
+      <div style="font-size: 8pt; font-weight: bold; width: 40mm; margin-left: 3mm; align-self: center; font-size: 10pt; text-align: center;">
+        <span id="nid-label">{{'SEARCH_PATIENT_NID' | translate}}:</span> {{patient.identifier}}
+      </div>
+      <div id="personal-data-conteiner">
+        <div id="first-line-personal-data" class="personal-data">Nº do Livro TARV: ______, Pág: _____, Linha: _____</div>
+        <div class="personal-data">{{'SEARCH_PATIENT_FIRSTNAME' | translate}}: {{patient.fullName}}</div>
+        <div class="personal-data">{{'PATIENT_INFO_OTHERS' | translate}}: {{patient['Numero de Telefone 1']}} / {{patient['Numero de Telefone 2']}}</div>
+        <div class="personal-data">{{'PATIENT_INFO_ADDRESS' | translate}}: {{patient.address.display}}</div>
```

Comment:
Address is not showing.

![image](https://user-images.githubusercontent.com/1554715/31884157-8d210fd2-b7ec-11e7-9776-0583a5c10c3f.png)